node/cn/filters/api_kaia_filter.go

```diff
@@ -77,24 +72,20 @@ type KaiaFilterAPI struct {
 	timeout time.Duration
 }
 
-// NewKaiaFilterAPI returns a new KaiaFilterAPI instance.
-func NewKaiaFilterAPI(backend Backend) *KaiaFilterAPI {
-	api := &KaiaFilterAPI{
+func NewFilterAPI(backend Backend) *FilterAPI {
```

Comment:
```suggestion
func NewFilterAPI(backend Backend, enablePendingTransaction bool) *FilterAPI {
```
to allow "newPendingTransaction" only in auction_subscribe, and disallow in kaia_subscribe/eth_subscribe.

Reply:
Thanks. [e6b4ccf](https://github.com/kaiachain/kaia/pull/466/commits/e6b4ccff374aa47582ebc5da97e26ba402e3e6cc)